src/index.html

```diff
@@ -6,9 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Alegreya&family=Alegreya+Sans&family=Archivo:wght@100;600;700&family=Fira+Sans:wght@300;400;500&family=Noto+Sans:wght@300;400;500;600;700&family=Open+Sans:wght@300&family=Oxygen:wght@400;700&family=Roboto:wght@100;300;400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo image">
+        </a>
+      </div>
```

Comment:
it's a redundant `<div >` wrapper
you can make your link as a block element
```suggestion
        <a href="#home" class="logo">
          <img src="images/logo.png" alt="logo image">
        </a>
```